src/pytest_plugins/test_filler/test_filler.py

```diff
@@ -175,6 +190,10 @@ def pytest_configure(config):
             f"{Frontier.solc_min_version()}",
             returncode=pytest.ExitCode.USAGE_ERROR,
         )
+    if config.getoption("version_info"):
+        pytest.exit(
+            f"'Displaying framework version information'\n" f"> {get_framework_version()}\n"
+        )
```

Comment:
If we use: `pytest.exit(get_framework_version())`

We get:
```
➜ fill --version-info
Exit:
execution-spec-tests v2.1.0-c5a62ea
```

I figured it would be nice to have a reason for the exit as there is no way of silencing the `"Exit:\n"`